articles/install-guide/vs-2017.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Installing the Q# development environment for VS 2017
+author: chgranad
+uid: microsoft.quantum.install.vs-2017
+ms.author: christopher.granade@microsoft.com
+ms.date: 7/31/2018
+ms.topic: article
+---
+
+# Installing the Quantum Development Kit for Visual Studio 2017 #
+
+## Installing Visual Studio 2017 (Windows 10 only)
+
+> [!NOTE]
+> The Microsoft Quantum Development Kit does not support Visual Studio for Mac.
+> To install on macOS, please follow the instructions for [using Visual Studio Code](xref:microsoft.quantum.install.vs-code).
+
+If you do not have Visual Studio installed, you can download Visual Studio 2017 Community Edition for free.
+1. Go to the [Visual Studio download page](https://www.visualstudio.com/downloads/).
+1. Click on the Visual Studio Community **Free download** button.
+2. Navigate to your browser's download folder and double click on the executable file whose name begins with **vs_community**. The file name will contain a sequence of numbers that varies.
+3. _**Important!**_ &nbsp;When you are presented with the option to select the tools for specific workloads, check the box for **.NET Core cross-platform development**.
+   If there are any other workloads you would like to install, you can select them as well at this step.
+4. After selecting your workloads, click **Install** to complete the installation.
+5. Once you have installed Visual Studio 2017, download the [Microsoft Quantum Development Kit for Visual Studio](https://marketplace.visualstudio.com/items?itemName=quantum.DevKit) extension from the Visual Studio Marketplace.
+   Double-click the VSIX once it is downloaded to install the extension.
+
+## Building the Quantum Samples and Libraries
```

Comment:
This is not part of the installation, so I wouldn't include it in the "Installing the QDK" section.

I would make this a new seccion:
\# Validating the Quantum Development Kit installation

Reply:
Fair point, I'm definitely happy to split that out. Many users have been confused by the inclusion of the validation step as a part of the installation guide, after all.